src/services/RepositoryService.js

```diff
@@ -1,35 +1,33 @@
-import api from '~config/api';
-
-export const createRepository = async values => {
-  const response = await api.post('/repositories', values);
-  if (response.ok) {
-    return response.data;
-  }
-  throw response;
-};
-
-export const addMemberToOrg = async values => {
-  const response = await api.post(`/organization/${values.username}`);
-  if (response.ok) {
-    return response.data;
-  }
-  throw response;
-};
-
-export const addOwnerToRepo = async values => {
-  const response = await api.post(`/repositories/${values.repository}/codeowners/`, {
+import { create } from 'apisauce';
+
+import api from '../config/api';
+
+/* export const createRepository = values => api.post('/repositories', values);
+
+export const addMemberToOrg = values => api.post(`/organization/${values.username}`);
+
+export const addOwnerToRepo = values =>
+  api.post(`/repositories/${values.repository}/codeowners/`, {
     codeowners: values.owners.split(',')
   });
-  if (response.ok) {
-    return response.data;
-  }
-  throw response;
-};
-
-export const getRepositories = async () => {
-  const response = await api.get('/repositories?limit=100&page=2');
-  if (response.ok) {
-    return response.data;
-  }
-  throw response;
-};
+*/
+
+export const getRepositories = () => api.get('/repositories?limit=100&page=2');
+
+export const createRepository = values =>
+  create({
+    baseURL: 'https://jsonplaceholder.typicode.com',
+    timeout: 55000
+  }).post('/posts', values);
+
+export const addMemberToOrg = values =>
```

Comment:
if you are going to mock de api, change the whole base url instead of changin for each function

Reply:
Move base URL to .env file